storage-node/src/cache/policy/mod.rs

```diff
@@ -50,13 +50,19 @@ impl DataStoreCacheValue for ParpulseDataStoreCacheValue {
 ///
 /// There are different cache policies for the data store cache, such as LRU, LRU-K, etc. See
 /// other files in this module for more details.
-pub trait DataStoreCache {
+pub trait DataStoreReplacer: Send + Sync {
```

Comment:
I'm also wondering if there is any use case for the rest of the methods in this trait...

Reply:
We can refine it in the future!